crmprtd/__init__.py

```diff
@@ -65,6 +68,54 @@ def __exit__(self, *args):
         self.run_time = self.end - self.start
 
 
+def common_script_arguments(parser):
```

Comment:
Can we exclude this function from our test coverage calculation? This is all standard library code and I don't think that there's value to testing it. Or just write a simple test function that passes it a parser, and checks that it gets a parser object back...